.travis.yml

```diff
@@ -16,13 +16,13 @@ script:
     - 'npm run test'
     - 'npm run build'
 # specify a job to run 
-deploy:
-    on:
-        branch: main
-        tags: false
-    skip_cleanup: true
-    provider: npm
-    email: $NPM_EMAIL_ADDRESS
-    api_key: $NPM_API_KEY
+# deploy:
```

Comment:
why not deploy?

Reply:
I haven't figured out the scripts to run for travis to do this yet. There are some quirkns in how this builds. I'm going to deploy from the command line and set this up later

Reply:
Specifically we are using `add-js-extension` package as a workaround to get the build to work properly. I manually check everything after it runs. I don't trust enough yet to automate with Travis.

Reply:
I have updated travis for an attempted auto deploy. I will manually confirm everything works if it works.